cmd/serve/dlna/data/data.go

```diff
@@ -2,3 +2,35 @@
 // The "go:generate" directive compiles static assets by running assets_generate.go
 
 package data
+
+import (
+	"io/ioutil"
+	"text/template"
+
+	"github.com/ncw/rclone/fs"
```

Comment:
This should be "github.com/rclone/rclone/fs" - that is why the CI is failing.

Reply:
Ahhh that makes sense.  :-)